mis_builder/models/aep.py

```diff
@@ -328,7 +328,7 @@ def get_aml_domain_for_dates(self, date_from, date_to,
         return expression.normalize_domain(domain)
 
     def do_queries(self, date_from, date_to, period_from, period_to,
-                   target_move):
+                   target_move, additional_move_line_filter=[]):
```

Comment:
In the link is all explain. Multiple calls to the same method will keep in the same variable what you append on each call.